project_timesheet_time_control/models/account_analytic_line.py

```diff
@@ -9,6 +9,7 @@
 
 class AccountAnalyticLine(models.Model):
     _inherit = 'account.analytic.line'
+    _order = 'date_time desc'
```

Comment:
Only this field? Shouldn't be more?

Reply:
The only field that I have seen that the account.analytic.line is ordered for is 'date' field field, therefore, I have only set it.

Reply:
But what about `id`? I think we should add it